### ISSUE TITLE
Check alt tags for images

### ISSUE THREAD

**Issue by LeandraH** [digitalfabrik/integreat-app#2535]
**Is your feature request related to a problem? Please describe.**
We should have an alt tag for all images that we show, otherwise a screenreader might e.g. read out the server address that the image is saved at. If the image doesn't contain useful information, we should have an empty string as alt tag.

**Describe the solution you'd like**
- If an image in the remote content doesn't have an alt tag, we should give it an empty string.
- For our icons on buttons, e.g. the search one, we should have an empty alt tag and handle the description of the action in the button around it, so a screenreader should read "search" for the button and not "magnifying glass" for the icon.
- If we have other icons that are not buttons (I can't think of any), we should think about whether they have a function that we want a screenreader to say. Maybe testing with a screenreader would be a good idea for that.

**Comment by svenseeberg** [comment]
https://www.w3.org/WAI/tutorials/images/decorative/ provides more context.

**Comment by svenseeberg** [comment]
> * Maybe testing with a screenreader would be a good idea for that.

I just tested Pericles and did not have any problems in the web app.
